app/views/page/show.html.erb

```diff
@@ -18,75 +18,103 @@
         <% when 'PageHrComponent' %>
-          <hr class="flex-fill border border-base-lighter bg-base-lightest dm-section-line margin-top-0 margin-bottom-3 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
+          <div class="grid-container">
+            <hr class="flex-fill border border-base-lighter bg-base-lightest dm-section-line margin-top-0 margin-bottom-3 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
+          </div>
 
         <%# Accordion %>
         <% when 'PageAccordionComponent' %>
           <% accordion_ctr += 1 %>
-
-          <div id="accordion_anchor_<%= accordion_ctr %>"
-               class="page-accordion-component margin-bottom-3<%= ' margin-bottom-0' if last_component == index %> <%= page_narrow_classes %>"
-          >
-            <div class="usa-accordion<%= ' usa-accordion--bordered' if component.has_border %>">
-              <h2 class="usa-accordion__heading">
-                <button class="usa-accordion__button font-sans-sm"
-                        aria-expanded="false"
-                        aria-controls="a<%= component.id %>">
-                  <%= component.title %>
-                </button>
-              </h2>
-              <div id="a<%= component.id %>" class="usa-accordion__content usa-prose">
-                <%= component.text.html_safe if component.text.present? %>
+          <div class="grid-container">
+            <div id="accordion_anchor_<%= accordion_ctr %>"
+                class="page-accordion-component margin-bottom-3<%= ' margin-bottom-0' if last_component == index %> <%= page_narrow_classes %>"
+            >
+              <div class="usa-accordion<%= ' usa-accordion--bordered' if component.has_border %>">
+                <h2 class="usa-accordion__heading">
+                  <button class="usa-accordion__button font-sans-sm"
+                          aria-expanded="false"
+                          aria-controls="a<%= component.id %>">
+                    <%= component.title %>
+                  </button>
+                </h2>
+                <div id="a<%= component.id %>" class="usa-accordion__content usa-prose">
+                  <%= component.text.html_safe if component.text.present? %>
+                </div>
               </div>
             </div>
           </div>
 
         <%# CTA %>
         <% when 'PageCtaComponent' %>
-          <div class="grid-row margin-bottom-3 <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
-            <div class="font-sans-lg grid-col-12 margin-bottom-2">
-              <span><%= component.cta_text %></span>
+          <% if component.has_background_color %>
+            <div class="bg-gray-90">
+              <div class="grid-container">
+                <div class="display-flex flex-align-center flex-justify-center" <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
```

Comment:
```suggestion
                <div class="display-flex flex-align-center flex-justify-center <%= page_narrow_classes %><%= ' margin-bottom-0' if last_component === index %>">
```

Oops. A small syntax error here.  We're not doing much with the Narrow style classes here, but we want them to apply in the future.

Reply:
ah good eye!